modules/meeting/app/services/meetings/ical_helpers.rb

```diff
@@ -67,7 +67,7 @@ def add_attendees(event, meeting)
     end
 
     def ical_uid(suffix)
-      "#{Setting.app_title}-#{Setting.host_name}-#{suffix}".dasherize
+      Digest::SHA256.hexdigest "#{Setting.app_title}-#{Setting.host_name}-#{suffix}"
```

Comment:
This will change the identifier of all existing meetings. Have we checked what the calendars do in this case or if it will duplicate calendar entries?